cores/esp32/esp32-hal-periman.h

```diff
@@ -14,6 +14,8 @@ extern "C"
 #include <stdbool.h>
 #include <stddef.h>
 
+#define perimanDetachPin(p) perimanSetPinBus(p, ESP32_BUS_TYPE_INIT, NULL, -1, -1)
```

Comment:
maybe `perimanClearPinBus` is a better name? it's more in line with `perimanSetPinBus`